src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -80,6 +81,119 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
             }
         }
 
+        /// <summary>
+        /// Endpoint for deleting delegated rules between parties
+        /// </summary>
+        /// <response code="201" cref="List{Rule}">Deleted</response>
+        /// <response code="206" cref="List{Rule}">Partial Content</response>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpPost]
+        [Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
+        [Route("authorization/api/v1/[controller]/DeleteRules")]
+        public async Task<ActionResult> DeleteRule([FromBody] List<RequestToDelete> rulesToDelete)
+        {
+            if (rulesToDelete == null || rulesToDelete.Count < 1)
+            {
+                return BadRequest("Missing rulesToDelete in body");
+            }
+
+            if (rulesToDelete.Any(r => r.RuleIds == null || r.RuleIds.Count == 0))
+            {
+                return BadRequest("Not all RequestToDelete has RuleId");
+            }
+
+            if (rulesToDelete.GroupBy(x => PolicyHelper.GetAltinnAppDelegationPolicyPath(x.PolicyMatch)).Any(g => g.Count() > 1))
```

Comment:
The `PolicyHelper.GetAltinnAppDelegationPolicyPath` method is able to throw exceptions that will ruin our attempt at providing a nice error message in return unless handled.

Reply:
Added a try catch and returns another message when the policy path can not be created for all requests